src/Octoshift/GithubApi.cs

```diff
@@ -696,8 +670,7 @@ public virtual async Task<string> GetUserId(string login)
             };
 
             // TODO: Add retry logic here, but need to inspect the actual error message and differentiate between transient failure vs user doesn't exist (only retry on failure)
-            var response = await _client.PostAsync(url, payload);
-            var data = JObject.Parse(response);
+            var data = await _client.PostGraphQLAsync(url, payload);
 
             return data["data"]["user"].Any() ? (string)data["data"]["user"]["id"] : null;
```

Comment:
A missing user triggers a GraphQL error, which now causes `PostGraphQLAsync` to throw, so this ternary is no longer needed. If we get to this point, there is a user.